src/input/index.test.js

```diff
@@ -6,13 +6,7 @@ import toJson from 'enzyme-to-json'
 
 import Input from './index'
 
-test('renders tags', t => {
-  const tags = [{ _id: 'i1', label: 'l1' }, { _id: 'i2', label: 'l2' }]
-  const wrapper = toJson(shallow(<Input tags={tags} />))
-  t.snapshot(wrapper)
-})
-
-test('renders input when no tags are passed', t => {
+test('renders input', t => {
```

Comment:
We'd need the existing tests to pass as is; again goes back to the opt-in based designs. So removing them completely defeats their purpose. We'd need new tests for the added features too.

Reply:
Since i renamed the Input to Tags, and the Input now only renders the Input. so the tests have been changed but the outputted html, css in still the same. 
I cant reuse the Input in two different locations without putting it in a separate component.